linera-execution/src/system.rs

```diff
@@ -652,10 +644,73 @@ where
             PublishDataBlob { blob_hash } => {
                 self.blob_published(&BlobId::new(blob_hash, BlobType::Data))?;
             }
+            PublishCommitteeBlob { blob_hash } => {
+                self.blob_published(&BlobId::new(blob_hash, BlobType::Committee))?;
+            }
             ReadBlob { blob_id } => {
                 self.read_blob_content(blob_id).await?;
                 self.blob_used(Some(txn_tracker), blob_id).await?;
             }
+            ProcessNewEpoch(epoch) => {
+                let chain_next_epoch = self.epoch.get().expect("chain is active").try_add_one()?;
+                ensure!(
+                    epoch == chain_next_epoch,
+                    SystemExecutionError::InvalidCommitteeCreation
```

Comment:
Maybe add a `String` argument do the error for a bit more context? Or a separate error.

Reply:
I added more detail to it in https://github.com/linera-io/linera-protocol/pull/3432/commits/0af786dbf224d42661039c0d74e09dae7979a32d.